LoopKitTests/DoseStoreTests.swift

```diff
@@ -750,3 +750,180 @@ class DoseStoreQueryTests: PersistenceControllerTestCase {
     }
     
 }
+
+class DoseStoreEffectTests: PersistenceControllerTestCase {
```

Comment:
Adding tests where doses are injected into the dose store, and the output insulin effect is verified.